src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2491,6 +2495,10 @@ theorem neg_neg {x : BitVec w} : - - x = x := by
   · simp [h]
   · simp [bv_toNat, h]
 
+@[simp]
```

Comment:
Note that [Int.neg_inj](https://leanprover-community.github.io/mathlib4_docs/Init/Data/Int/Lemmas.html#Int.neg_inj) does not use @[simp], I asked about this [here](https://leanprover.zulipchat.com/#narrow/channel/113489-new-members/topic/When.20to.20use.20.40.5Bsimp.5D) but didn't get an answer. I'm not sure if there is some deep reason as to why this should be avoided or if it's just missing.

Reply:
I think this is a good question, CC @kim-em

Reply:
After some discussion with @TwoFX this seems sensible and we should probably consider setting the `simp` attribute for the respective `Int` lemma in core.